man/systemd.generator.xml

```diff
@@ -101,11 +101,14 @@
 
     <para>After installing new generators or updating the
     configuration, <command>systemctl daemon-reload</command> may be
-    executed. This will delete the previous configuration created by
-    generators, re-run all generators, and cause
-    <command>systemd</command> to reload units from disk. See
-    <citerefentry><refentrytitle>systemctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>
-    for more information.
+    executed. With respect to generators, this will delete the previous
+    configuration created by them, re-run all generators, and cause
+    <command>systemd</command> to reload units from disk. Generated units
+    may have their configurations overridden by provided unit files,
+    directories, or templates placed in the /etc/systemd/system directory,
+    as explained in <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry.
```

Comment:
missing `>`.

Reply:
The important part is this one new sentence: "Generated units may have their configurations overridden by provided unit files, directories, or templates placed in the /etc/systemd/system directory, ...". But the phrasing is awkward, and this doesn't seem to be the right place to put this — this paragraph discusses reloading, and overriding is an unrelated issue. The other changes that you make are better left out.

What about just adding a new paragraph (as paragraph two, before the one that starts with "Generators are loaded from"), that contains just one sentence that describes how to override unit files generated by generators?